pkg/event/target/webhook.go

```diff
@@ -49,6 +54,14 @@ func (w WebhookArgs) Validate() error {
 	if w.Endpoint.IsEmpty() {
 		return errors.New("endpoint empty")
 	}
+	if w.QueueDir != "" {
+		if !filepath.IsAbs(w.QueueDir) {
+			return errors.New("queueDir path should be absolute")
+		}
+	}
+	if w.QueueLimit > 10000 {
```

Comment:
Can `maxLimit` defined in https://github.com/minio/minio/blob/master/pkg/event/target/queuestore.go#L31 be used here?

Reply:
Done